src/main/java/org/apache/maven/shared/incremental/IncrementalBuildHelper.java

```diff
@@ -61,17 +69,28 @@
     private MavenProject mavenProject;
 
     /**
-     * Used for detecting changes between the Mojo execution.
+     * Used for detecting changes in the output directory between the Mojo execution.
```

Comment:
something's off with "between the Mojo execution". possibly should be "between Mojo executions" or perhaps something else is meant?